### ISSUE TITLE
"Could not find artifact com.sun:tools:jar:1.7.0" with Java 10

### ISSUE THREAD

**Issue by simonDeHero** [travis-ci/travis-ci#9926]
Hi,
when i specify `oraclejdk10` or `openjdk10`, i get the following error:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check (validate) on project X: Execution validate of goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check failed: Plugin org.apache.maven.plugins:maven-checkstyle-plugin:2.17 or one of its dependencies could not be resolved: Could not find artifact com.sun:tools:jar:1.7.0 at specified path /home/travis/oraclejdk10/../lib/tools.jar -> [Help 1]
```

thx for looking into it!
bye
Simon

**Comment by simonDeHero** [comment]
it is actually an issue of the maven-checkstyle-plugin. [here](https://issues.jboss.org/browse/WELD-2149) a nice explanation.

**Comment by simonDeHero** [comment]
just for completeness, i resolved it using version 3.0.0 of the plugin and 
```
<dependency>
    <groupId>com.puppycrawl.tools</groupId>
    <artifactId>checkstyle</artifactId>
    <version>8.11</version>
</dependency>
```
